fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_objectives.dm

```diff
@@ -376,9 +362,15 @@
 
 // EXPLANATION
 /datum/objective/bloodsucker/vassal/update_explanation_text()
-	explanation_text = "Guarantee the success of your Master's mission!"
+	. = ..()
+	explanation_text = "Guarantee the success of your Master's mission! Their objectives are as follows: \n"
```

Comment:
This is stupid. Bloodsucker objectives are optional, they can just fuck around. Vassals don't need to care if the Bloodsucker doesnt care enough to tell them.

Reply:
Fair enough, I'll revert that